<launch>	
    
    

   <!-- Start the Dynamixel low-level driver manager with parameters -->
   <node name="dynamixel_manager" pkg="dynamixel_controllers"
      type="controller_manager.py" required="true" output="screen">
      <rosparam>
         namespace: pi_dynamixel_manager
         serial_ports:
            dynamixel_ax12:
               port_name: "/dev/sensors/ftdi_A400gac2"
               baud_rate: 1000000
               min_motor_id: 1
               max_motor_id: 10
               update_rate: 20
      </rosparam>
   </node>
   
    <!-- Load joint controller configuration from YAML file to parameter server -->
    
   <rosparam file="$(find robbie_bringup)/params/dynamixel.yaml" command="load"/>

   <!-- Start the head pan and tilt controllers -->
   <node name="dynamixel_controller_spawner_ax12" pkg="dynamixel_controllers"
   type="controller_spawner.py"
        args="--manager=pi_dynamixel_manager
              --port=dynamixel_ax12
              --type=simple
        head_pan_joint
        head_tilt_joint
        right_arm_wrist_yaw_joint
        right_arm_wrist_tilt_joint
        right_arm_gripper_joint
        left_arm_wrist_tilt_joint
        left_arm_wrist_yaw_joint
        left_arm_gripper_joint
        "
        output="screen" />

<remap from="/head_tilt_joint/set_speed" to="/head_tilt_mod_joint/set_speed" />
<!-- node to republish neck tilt with gearing-->
        
    <node name="neck" pkg="robbie_bringup" type="neck_tilt.py"/>

  
  
</launch>
